### PR TITLE
Add fill and line color docs

### PR DESCRIPTION
**Pull request by timmo001**
## Proposed change

Adds docs for home-assistant/frontend/pull/5224

## Type of change

- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).

## Additional information

- Link to parent pull request in the codebase: home-assistant/frontend/pull/5224

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_lovelace/sensor.markdown [hunk 1]
```diff
@@ -46,6 +46,10 @@ hours_to_show:
   description: Hours to show in graph
   type: integer
   default: 24
+graph:
```
Comment (frenck) [inline source/_lovelace/sensor.markdown]: ?
```suggestion
graph_color:
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_lovelace/sensor.markdown (hunk 1 above)
Comment (timmo001) [reply source/_lovelace/sensor.markdown]: Thanks @frenck ! :1st_place_medal:

**Review by frenck — APPROVED** [review]

**Comment by timmo001** [comment]
Attached PR closed

**Pull request closed without merging**